hbase-server/src/main/java/org/apache/hadoop/hbase/master/replication/ReplicationPeerManager.java

```diff
@@ -493,12 +499,35 @@ private void checkConfiguredWALEntryFilters(ReplicationPeerConfig peerConfig)
     }
   }
 
-  private void checkClusterKey(String clusterKey) throws DoNotRetryIOException {
+  private void checkClusterKeyForHBaseInterClusterReplicationEndpoint(String clusterKey)
+    throws DoNotRetryIOException {
     try {
       ZKConfig.validateClusterKey(clusterKey);
     } catch (IOException e) {
       throw new DoNotRetryIOException("Invalid cluster key: " + clusterKey, e);
     }
+    String peerClusterId = "";
```

Comment:
You are fetching the currentClusterId right? Why name it as peerClusterId? I think the argument to this method is peerClusterKey.

Reply:
No, this fetch the peer cluster id.